data/sql/updates/pending_db_world/rev_1569250786242020759.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1569250786242020759');
+
+ALTER TABLE `creature_addon` ADD COLUMN `isLarge` TINYINT(3) UNSIGNED NOT NULL DEFAULT 0 AFTER `emote`;
+ALTER TABLE `creature_template_addon` ADD COLUMN `isLarge` TINYINT(3) UNSIGNED NOT NULL DEFAULT 0 AFTER `emote`;
```

Comment:
why tinyint? It can be boolean

Reply:
There is no built-in data type "boolean", it is a synonym for tinyint(1). But you're right, tinyint(1) suffices here. I have pushed a new commit.

Reply:
yeah, so confusing, i remember reading this on google like 4 times xD